### DISCUSSION TITLE
How to Install and run doctr pytorch build despite having tensorflow installed in my environment

### DISCUSSION THREAD

**Discussion by prithivi1** [mindee/doctr#1729] [Q&A] [answered]
I Need to install doctr package in my python environment, where both tensorflow and pytorch is installed. I would like to use the pytorch version. How can set that. 

By default it is loading the tensorflow version since it is checked at first in  models.utils.init.py
```
from doctr.file_utils import is_tf_available, is_torch_available

if is_tf_available():
    from .tensorflow import *
elif is_torch_available():
    from .pytorch import *  # type: ignore[assignment]
```

How can I skip this and import pytorch

**Comment by felixdittrich92** [comment, marked as answer]
Hi @prithivi1 :wave:,

You can set an env variable: 
(NOTE: set this on top of your code before importing any doctr components)

```
import os
os.environ["USE_TORCH"] = "1"
```

or run your script with it:
```
USE_TORCH=1 python3 path/to/your_script.py
```

Same works with TensorFlow where USE_TF=0 would disable it and =1 enable :)

**Comment by prithivi1** [comment]
Thanks @felixdittrich92
